lms/djangoapps/certificates/generation_handler.py

```diff
@@ -456,56 +393,14 @@ def generate_user_certificates(student, course_key, insecure=False, generation_m
         forced_grade - a string indicating to replace grade parameter. if present grading
                        will be skipped.
     """
-    if can_generate_certificate_task(student, course_key):
-        # Note that this will launch an asynchronous task, and so cannot return the certificate status. This is a
-        # change from the older certificate code that tries to immediately create a cert.
-        log.info(f'{course_key} is using V2 certificates. Attempt will be made to regenerate a V2 certificate for user '
-                 f'{student.id}.')
-        return generate_certificate_task(student, course_key)
-
-    beta_testers_queryset = list_with_level(course_key, 'beta')
-    if beta_testers_queryset.filter(username=student.username):
-        log.info(f"Canceling Certificate Generation task for user {student.id} : {course_key}. User is a Beta Tester.")
-        return
-
-    xqueue = XQueueCertInterface()
-    if insecure:
-        xqueue.use_https = False
-
-    course_overview = get_course_overview_or_none(course_key)
-    if not course_overview:
-        log.info(f"Canceling certificate generation for user {student.id} : {course_key} due to a missing course "
-                 f"overview.")
-        return
-
-    generate_pdf = not has_html_certificates_enabled(course_overview)
-
-    cert = xqueue.add_cert(
-        student,
-        course_key,
-        generate_pdf=generate_pdf,
-        forced_grade=forced_grade
-    )
+    # Note that this will launch an asynchronous task, and so cannot return the certificate status. This is a
+    # change from the older certificate code that tries to immediately create a cert.
+    log.info(f'{course_key} is using V2 certificates. Attempt will be made to regenerate a V2 certificate for user '
+             f'{student.id}.')
+    return generate_certificate_task(student, course_key)
 
-    log.info(f"Queued Certificate Generation task for {student.id} : {course_key}")
 
-    # If cert_status is not present in certificate valid_statuses (for example unverified) then
-    # add_cert returns None and raises AttributeError while accessing cert attributes.
-    if cert is None:
-        return
-
-    if CertificateStatuses.is_passing_status(cert.status):
-        emit_certificate_event('created', student, course_key, course_overview, {
-            'user_id': student.id,
-            'course_id': str(course_key),
-            'certificate_id': cert.verify_uuid,
-            'enrollment_mode': cert.mode,
-            'generation_mode': generation_mode
-        })
-    return cert.status
-
-
-def regenerate_user_certificates(student, course_key, forced_grade=None, template_file=None, insecure=False):
+def regenerate_user_certificates(student, course_key, forced_grade=None, template_file=None, insecure=False):  # pylint: disable=unused-argument
```

Comment:
Could we remove the unused argument rather than keeping it?

Reply:
Since this method was written for V1 of course certificates, I'll be removing it entirely in a future PR (part of my next ticket). Any callers will be switched over to a V2 method. Stay tuned...